### ISSUE TITLE
Trigger Warnings - Behavior

### ISSUE THREAD

**Issue by ghost** [w3c/personalization-semantics#89]
Define semantics used to enforce code of conduct or acceptable use policies between users or within the context of interactive content.

**Comment by thaddeus-cambron** [comment]
Not necessarily. A trigger is often found in a group in which a user is already comfortable with and is an active member. Another user or site owner may post or comment a trigger unintentionally. Triggers are highly personalized and subjective.
